package.json

```
@@ -19055,13 +19055,19 @@
     },
     "node-linux-x64": {
       "version": "*"
+    },   
+    "node-red-node-rdk-tools": {
```

Biome (2.4.4)

[error] The key node-red-node-rdk-tools was already declared.

(lint/suspicious/noDuplicateObjectKeys)